### PR TITLE
fix: improve the structure of statement

### PR DESCRIPTION
**Pull request by yin1999**
### Description

improve the structure of statement

### CODE REVIEW

File: files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md [hunk 1]
```diff
@@ -55,7 +55,10 @@ With these techniques, you can configure your server to respond correctly for bo
 
 ## Deciding the case
 
-This is a very subjective topic — it could be considered a [bikeshedding](https://bikeshed.com/) issue. If you wish to read deeper, please see some of the [many](https://www.netlify.com/blog/2020/03/26/how-to-set-up-netlify-dns-custom-domains-cname-and-a-records/#options-for-bare-domains) [articles](https://www.wpbeginner.com/beginners-guide/www-vs-non-www-which-is-better-for-wordpress-seo/) on the subject.
+This is a very subjective topic — it could be considered a [bikeshedding](https://bikeshed.com/) issue. If you wish to read deeper, here are some resources:
+
+- [Options for bare domains](https://www.netlify.com/blog/2020/03/26/how-to-set-up-netlify-dns-custom-domains-cname-and-a-records/#options-for-bare-domains) (2020)
+- [WWW vs non-WWW – Which Is Better for WordPress SEO?](https://www.wpbeginner.com/beginners-guide/www-vs-non-www-which-is-better-for-wordpress-seo/) (2023)
```
Comment (hamishwillee) [inline files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md:61]: AFAIK There is no standard format for references on MDN, but IMO the source of the link is probably more relevant information than the date.

```suggestion
- [Options for bare domains](https://www.netlify.com/blog/2020/03/26/how-to-set-up-netlify-dns-custom-domains-cname-and-a-records/#options-for-bare-domains) (www.netlify.com)
- [WWW vs non-WWW – Which Is Better for WordPress SEO?](https://www.wpbeginner.com/beginners-guide/www-vs-non-www-which-is-better-for-wordpress-seo/) (www.wpbeginner.com)
```

Comment (yin1999) [reply files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md:61]: Thanks for the advice @hamishwillee. Should we use the format discussed [here](https://github.com/orgs/mdn/discussions/339#discussioncomment-5173159):

```suggestion
- [Options for bare domains](https://www.netlify.com/blog/2020/03/26/how-to-set-up-netlify-dns-custom-domains-cname-and-a-records/#options-for-bare-domains) on www.netlify.com (2020)
- [WWW vs non-WWW – Which Is Better for WordPress SEO?](https://www.wpbeginner.com/beginners-guide/www-vs-non-www-which-is-better-for-wordpress-seo/) on www.wpbeginner.com (2023)
```

Comment (hamishwillee) [reply files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md:61]: @dipikabh Can you please point me to the final guideline on how to mark up reference? FWIW I'm not sure about dates  - if they are recent that is a good hint they might be "relevant" but if they are old it is never clear if they are out of date or just timeless ... but old dates always make the surround content look old.

Comment (hamishwillee) [reply files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md:61]: Ignore that. Found the section. Yes, I think this is correct to style guide.

**Review by hamishwillee — APPROVED** [review]
THanks @yin1999

**Pull request merged**